docs/reference/2-ods-api/client-developers-guide/error-response-knowledge-base.md

```diff
@@ -277,24 +294,25 @@ The Ed-Fi Data Model prescribes a maximum string length in many cases. The
 application code needs to truncate strings to fit, if they are larger than the
 data model's allowed length.
 
-##### Example: `codeValue` is longer than 50 characters
+<details>
+<summary>Example: `codeValue` is longer than 50 characters</summary>
 
-Http Request
+HTTP Request
 
-```
+```text
 POST https://api.ed-fi.org:443/v7.2/api/data/v3/ed-fi/absenceEventCategoryDescriptors
 Authorization: bearer 6f5bb488a65948b5b847b561b23e
 Content-Type: application/json
 
 {
-  "codeValue": "Bereavementddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd",  
+  "codeValue": "Bereavementddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd",
   "shortDescription": "Bereavement",
```

Comment:
Just making sure this character will not cause any trouble. Looks like a line break?